news/models.py

```diff
@@ -0,0 +1,39 @@
+from django.conf import settings
+from django.core.paginator import EmptyPage, PageNotAnInteger, Paginator
+from django.utils.translation import ugettext_lazy as _
+from wagtail.wagtailcore.models import Page
+
+from .news import get_news_feeds
+
+
+class NewsIndexPage(Page):
+
+    @property
+    def news_list(self):
+        return get_news_feeds()
+
+    class Meta:
+        verbose_name = _('News index')
+
+    def get_context(self, request, *args, **kwargs):
+        context = super(NewsIndexPage, self).get_context(request, *args, **kwargs)
+
+        news_list = self.news_list
+
+        # Pagination
+        page = request.GET.get('page')
+        page_size = 10
+        if hasattr(settings, 'NEWS_PAGINATION_PER_PAGE'):
```

Comment:
I'm confused. Wouldn't a single getattr(settings, 'NEWS_PAGINATION_PER_PAGE', 10) work here?

Reply:
You're right. This is now fixed.